app/steps/ui/applicant/alias/index.js

```diff
@@ -0,0 +1,8 @@
+const ValidationStep = require('app/core/steps/ValidationStep');
+
+module.exports = class ApplicantAlias extends ValidationStep {
```

Comment:
Can you export this class at the bottom of the file, see `app/steps/ui/applicant/nameasonwill/index.js`